containers/Ideas/Idea.js

```diff
@@ -20,6 +21,9 @@ export const Idea = (props) => {
   const [deleteModalVisible, setDeleteModalVisible] = useState(false);
   const [processing, setProcessing] = useState(false);
   const [comments, setComments] = useState([]);
+  const [objectToComment, setObjectToComment] = useState({'contentType': idea.content_type, 'pk': idea.pk});
```

Comment:
i was a little confused by the name of this state variable. Not sure of a better alternative though?
Maybe `commentMeta` or something?

Reply:
Yeah, totally also not happy with this name! It holds the content_type and pk of the object that is currently commented on, so either the idea or a comment... maybe commentOn? What do you think @khamui?

Reply:
hmm..yes could be. Or something like `commentContext`, `parentOfComment`...uff i am bad at this, looking at `objectToComment` makes kind of sense now..😬

or why not `contentTypeOfComment`?

Reply:
@khamui What do you think of contentObjectOfComment? That would make sense in the way, that in django, the comment model does have a [content_object](https://github.com/liqd/adhocracy4/blob/db606bfb85f5af3e6c0cb13b1ce723adba681091/adhocracy4/comments/models.py#L22) that hold the content_type and pk of the commented object. So actually just what we have here..